docs/production-candig.md

```diff
@@ -0,0 +1,122 @@
+# Guide to CanDIG production deployments
+
+Apart from the basic steps in the [CanDIGv2 Install Guide](install-candig.md) to get the candig stack up and running, there are additional settings and security recommendations that need to set up in a production level environment. We provide the following as general advice, but it is important for all CanDIG deployers to also consult with their institutional infrastructure security personnel to ensure that their deployment meets the necessary level of data security.
+
+## Reverse Proxy & Firewall
+
+It is essential to setup a reverse proxy and firewall so that only specific ports are open to the internet. The software used for this is up to the deployer and is considered outside of the CanDIG stack. 
+
+Basically, the only ports that should be available are to tyk (443) and keycloak (80).
```

Comment:
This might be me being very dumb, but isn't keycloak 8080? or do we redirect to that?

Reply:
I have seen different things in differnt docs and I don't understand the difference. @OrdiNeu ?

Reply:
Would be good to have confirmation whether or not this is correct